Tests/Creation/Frame/Content/String/ID3FrameFromBooleanContentCreatorTest.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ID3FrameFromStringContentCreatorTest.swift
+//
+//  Created by Nolaine Crusher on 03/31/2020.
+//  2018 Fabrizio Duroni.
+//
+
+import XCTest
+@testable import ID3TagEditor
+
+class ID3BooleanFrameCreatorTest: XCTestCase {
```

Comment:
I should really have this test in here, but I'm struggling with how to write it.